data.sql

```diff
@@ -46,4 +46,41 @@ SET species_id = (
         ELSE
             (SELECT id FROM species WHERE name = 'Pokemon')
     END
-);
\ No newline at end of file
+);
+
+INSERT INTO vets (name, age, date_of_graduation) 
+VALUES 
+  ('William Tatcher', 45, '2000-04-23'),
+  ('Maisy Smith', 26, '2019-01-17'),
+  ('Stephanie Mendez', 64, '1981-05-04'),
+  ('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO specializations (vet_id, species_id) 
+VALUES
+  (1,1),
+  (3,1),
+  (3,2),
+  (4,2);
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit) 
```

Comment:
- [ ] Kindly fix your `INSERT` method to pass the correct query to the database, since it's necessary to mention which values are needed to insert. Like 

`INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24');`